docs/conda_windows_install.md

```diff
@@ -17,3 +17,71 @@ conda deactivate
 ```
 
 To use your environment, run the command "conda activate popsborder_env" or select the popsborder_env environment from your IDE (e.g. VS Code, Atom, PyCharm).
+
+## Adding Conda Environments to Jupyter Notebook
+
+ Conda allows you to create isolated environments with specific 
+ versions of Python and its packages. This prevents conflicts between 
+ different projects' dependencies.
+
+### Step 1: Create your conda environment.
+Inside of your conda based command line navigate inside your forked PopsBorder
+folder. Next use the following command
+```bat
+conda create -n PopsBorderEnv python=3.8
+```
+This creates your environment with python version 3.8.
+PopsBorderEnv is the name of your environment. Change as needed.
+
+### Step 2: Activate your environment.
+Using your newly created environment from the previous step, run
+```bat
+conda activate PopsBorderEnv
+```
+Now you have entered into your environment.
+
+### Step 3: Install pipenv into you environment.
+Now install pipenv into your environment, a packaging tool we utilize
+for our project.
+```bat
+pip install pipenv
+```
+
+### Step 4: Install dependencies using pipenv.
+To download dependencies and create a virtual environment from the Pipfile.lock use: 
+```bat
+pipenv install
+```
+Now that your virtual environment is created, navigate into it with
+```bat
+pipenv shell
+```
```

Comment:
Is this needed? (esp. in context of using conda install.) We want to minimize number of commands people need to run.

Reply:
After running some tests this does seem to be needed as pipenv installs the dependencies inside its virtual environment. The only other way would be putting pipenv run before the rest of the commands. But I can leave that up to you.

Reply:
I chose the option to reduce the number of commands like you said. I also tested this and ensured it worked on the notebook examples.